core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/DiagnosticLog.java

```diff
@@ -102,4 +121,14 @@ public Map<String, Object> getConfigurations() {
 
         return configurations;
     }
+
+    public LogLevel getPersona() {
```

Comment:
shall we use `getLogLevel`?

Reply:
Fixed with 5a88f68259a09bda7d564506434c02818d7e6c74